langbase/errors.py

```diff
@@ -62,56 +71,41 @@ def _make_message(
             status: HTTP status code
             error: Error response body
             message: Error message
-            endpoint: API endpoint that was called
-            request_id: Request ID from headers
+            endpoint: API endpoint that was called (unused, for compatibility)
+            request_id: Request ID from headers (unused, for compatibility)
 
         Returns:
             Formatted error message string
         """
-        # Extract the main error message
+        # Decide the base message or object to display
         if isinstance(error, dict) and "message" in error:
-            msg = error["message"]
-            if not isinstance(msg, str):
-                msg = str(msg)
-        elif error:
-            msg = str(error)
+            base_message: Any = error["message"]
+        elif error is not None:
+            base_message = error
         else:
-            msg = message
-
-        # Build comprehensive error message
-        parts = []
-
-        # Status line
-        if status:
-            status_text = STATUS_CODE_TO_MESSAGE.get(status, "Unknown Error")
-            parts.append(f"{status_text} ({status})")
-
-        # Error message
-        if msg:
-            parts.append(f"\n  Message: {msg}")
-
-        # API endpoint
-        if endpoint:
-            parts.append(f"\n  Endpoint: {endpoint}")
-
-        # Request ID
-        if request_id:
-            parts.append(f"\n  Request ID: {request_id}")
-
-        # Error details from response
-        if isinstance(error, dict):
-            if "code" in error:
-                parts.append(f"\n  Error Code: {error['code']}")
-            if "details" in error:
-                parts.append(f"\n  Details: {error['details']}")
-
-        # Documentation link
-        if status:
-            parts.append(
-                f"\n  Documentation: https://langbase.com/docs/errors/{status}"
-            )
+            base_message = message
+
+        # Convert dict/list to JSON string (double quotes), keep strings as-is
+        try:
```

Comment:
I do not follow what are we trying to do here? Why this dynamic import? What does pragma means here?